js/main.js

```diff
@@ -0,0 +1,391 @@
+				recipeIcons.html(iconString);		
+		
+				// append all card items
+				recipeImageContainer.append(recipeImage, recipeButton);
+				recipeContent.append(recipeSource, recipeName, recipeIcons);
+				recipeCard.append(recipeImageContainer, recipeContent);		
+				recipeCarousel.append(recipeCard);
+				
+			});
+	
+			// append recipe section
+			recipes.append(recipeHeading, recipeCarousel);
+			contentContainer.append(recipes);
+		
+			// recipe API carousel
+			$('#recipes .owl-carousel').owlCarousel({
+				margin: 20,
+				responsiveClass:true,
+			    responsive:{
+			        0:{
+			            items:1
+			        },
+			        575:{
+			            items:2
+			        },
+			        767:{
+			            items:3
+			        },
+			        1199:{
+			            items:4,
+			            loop:true
+			        }
+			    },
+				loop: true,
+				nav:true,
+				navText: ['',''],
+				dots: false,
+				lazyLoad: false,
+				autoplay: false,
+				navSpeed: 500
+			});
+			
+			// create menus section
+		    var menus = $('<section>').attr('id', 'menus').addClass('row');
+		    var menusHeading = $('<div>').addClass('col s12').html('<h2>Venture Out</h2>');
+		    var menusCarousel = $('<div>').addClass('owl-carousel owl-theme col s12');
+		    
+		    responseMenu[0].menuItems.forEach(function(i) {
+				
+				// create card
+				var menusCard = $('<div>').addClass('card');
+				
+				// create card image items
+				var menusImageContainer = $('<div>').addClass('card-image');
+				var menusImage = $('<div>').addClass('image');
+				menusImage.attr('style', 'background-image: url(' + i.image + ')');		
+				var menusButton = $('<a>').addClass('btn-floating btn-large halfway-fab cyan');
+				// update this when you update restaurant name from restaurant api
+				menusButton.attr("data-restaurant-name", i.restaurantChain);
+				menusButton.html('<i class="fal fa-map-marker-alt"></i>');
+				
+				// create card content items
+				var menusContent = $('<div>').addClass('card-content');		
+				var menusRestaurant = $('<p>').attr('id', 'restaurant');
+				menusRestaurant.text(i.restaurantChain);		
+				var menusName = $('<h3>').attr('id', 'menu-name');
+				menusName.text(i.title);		
+				var menusIcons = $('<div>').attr('id', 'icons');
+				menusIcons.html(iconString);		
+		
+				// append all card items
+				menusImageContainer.append(menusImage, menusButton);
+				menusContent.append(menusRestaurant, menusName, menusIcons);
+				menusCard.append(menusImageContainer, menusContent);		
+				menusCarousel.append(menusCard);
+				
+				// on menu item button click...
+				menusButton.on("click", function (event) {
+					window.location.href="https://www.google.com/maps/search/?api=1&query=" + i.restaurantChain;
+
+					event.preventDefault();
+					$("#map").attr("style", "display: block");
+					
+					var restaurantMap = $(this).attr("data-restaurant-name");
+					console.log({ restaurantMap });
+	
+					var mapURL =
```

Comment:
minor stylistic thing, you should have your `+` at the beginning of a line

it makes it much easier to read since all of the pluses will be lined up and you would be surprised how easy it is to miss a plus at the end when you modify code